src/base/NekRSSeparateDomainProblem.C

```diff
@@ -297,34 +269,15 @@ NekRSSeparateDomainProblem::sendBoundaryScalar02ToNek()
 
   auto & mesh = _nek_mesh->getMesh();
 
-  _console << "Sending scalar02 of " << *_toNekRS_scalar02 << " to NekRS boundary "
-           << Moose::stringify(_inlet_boundary) << std::endl;
-
-  for (unsigned int e = 0; e < _n_surface_elems; e++)
-    scalar02(e, *_toNekRS_scalar02);
-}
-
-void
-NekRSSeparateDomainProblem::sendBoundaryScalar03ToNek()
-{
-  auto & solution = _aux->solution();
-  auto sys_number = _aux->number();
-
-  if (_first)
-  {
-    _serialized_solution->init(_aux->sys().n_dofs(), false, SERIAL);
-    _first = false;
-  }
-
-  solution.localize(*_serialized_solution);
-
-  auto & mesh = _nek_mesh->getMesh();
+//  // check which scalar PP to pass to NekRS
+//  const PostprocessorValue * scalarValue = nullptr;
```

Comment:
Does this commented-out code need to be here?

Reply:
Thanks for pointing this out, I missed it.